bundles/ch.elexis.core.data/src/ch/elexis/core/data/activator/CoreHub.java

```diff
@@ -464,13 +484,83 @@ public static int getSystemLogLevel(){
 		return localCfg.get(Preferences.ABL_LOGLEVEL, Log.ERRORS);
 	}
 	
+	/**
+	 * Login: Anwender anmelden, passenden Mandanten anmelden. (Jeder Anwender
+	 * ist entweder selber ein Mandant oder ist einem Mandanten zugeordnet)
+	 * 
+	 * @param username
+	 *            Kurzname
+	 * @param password
+	 *            Passwort
+	 * @return <code>true</code> erfolgreich angemeldet, CoreHub.actUser
+	 *         gesetzt, else <code>false</code>
+	 * @since 3.1 queries {@link User}
+	 */
+	public static boolean login(final String username, final String password) {
```

Comment:
Javadoc bitte englisch und `@since 3.8`. Passwort bitte als char[] siehe https://stackoverflow.com/questions/8881291/why-is-char-preferred-over-string-for-passwords 

Javadoc `@throws LoginExcpetion`

Die Priority müssen wir hier noch diskutieren. Aus meiner Sicht sollte es anders rum sein, so dass die Elexis DB immer den fallback bildet. Evtl. müssen wir eine start-flag einführen.

Reply:
ok müssma bereden